pkg/generate/generate_controller.go

```diff
@@ -128,6 +137,9 @@ func NewController(
 		UpdateFunc: c.updateGenericResource,
 	})
 
+	c.nsLister = namespaces.Lister()
```

Comment:
If we create nsLister from `informersv1.NamespaceInformer`, let's remove dynamic informer (line 134-135). 
Besides, do we still need event handler for the namespace? Previously we used it as triggers to reconcile gr, now we handle it in the webhook.